### PR TITLE
aaplus: cross-compatible conan v1 & v2

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **lib/1.0**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by uilianries — APPROVED** [review]
LGTM

File: recipes/aaplus/all/conanfile.py [hunk 1]
```diff
@@ -56,44 +52,44 @@ def _compilers_minimum_version(self):
         }
 
     def validate(self):
-        if self.settings.compiler.get_safe("cppstd"):
-            tools.check_min_cppstd(self, "17")
+        if self.info.settings.compiler.cppstd:
```
Comment (prince-chrismc) [inline recipes/aaplus/all/conanfile.py:55]: Why is get safe no longer needed?

Comment (SpaceIm) [reply recipes/aaplus/all/conanfile.py:55]: There is no get_safe with self.info.options

Comment (SSE4) [reply recipes/aaplus/all/conanfile.py:55]: see https://github.com/conan-io/conan-center-index/pull/12006#discussion_r939180538

Comment (prince-chrismc) [reply recipes/aaplus/all/conanfile.py:55]: 🤢 Thanks for the details

**Review by prince-chrismc — APPROVED** [review]

File: recipes/aaplus/all/conanfile.py (hunk 1 above)
Comment (uilianries) [reply recipes/aaplus/all/conanfile.py:55]: Not yet: https://github.com/conan-io/conan/issues/11802

**Pull request merged**